tests/bot/utils/test_time.py

```diff
@@ -46,11 +46,11 @@ def test_humanize_delta_raises_for_invalid_max_units(self):
         for max_units in test_cases:
             with self.subTest(max_units=max_units), self.assertRaises(ValueError) as error:
                 time.humanize_delta(relativedelta(
-                    days=2, hours=2), 'hours', max_units)
+                    days=2, hours=2), "hours", max_units)
             self.assertEqual(str(error.exception),
-                             'max_units must be positive')
+                             "max_units must be positive")
```

Comment:
```suggestion
            self.assertEqual(
                str(error.exception),
                "max_units must be positive"
            )        
```

This format is a bit more readable

Reply:
Fixed by 7922313416d1f82147f6e79be45ed683d12db04a